packages/echarts/src/computecss.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Siemens AG
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+
+export default function getComputedCSSProperty(cssValue: string) {
```

Comment:
Library internal function we should take over the prefixing e.g

```ts
....getPropertyValue(`--theme-color-${cssValue}`);
```

the user can use it then with:

```
color: getComputedCSSProperty('neutral'),
```

Reply:
I have prefixed it as far as possible ....getPropertyValue(`--theme-${cssValue}`); 
We have --theme-color and --theme-chart in use